mfhdf/dumper/hdp.c

```diff
@@ -11,11 +11,28 @@
  * help@hdfgroup.org.                                                        *
  * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 
-#define HDP_MASTER
 #define VSET_INTERFACE
 #include "hdp.h"
 #include "local_nc.h" /* to use some definitions */
 
+/* Global Variables */
+
+/* indicates Vsets have been initialized for the current file */
+intn vinit_done = FALSE;
+
+/* hdp commands (stored as (value, name) pairs to keep them in sync) */
+typedef enum { HELP, LIST, DUMPSDS, DUMPRIG, DUMPVG, DUMPVD, DUMPGR, BAD_COMMAND } command_value_t;
+
+typedef struct command_t {
+    const command_value_t value;
+    const char           *name;
+} command_t;
```

Comment:
Tying the enum value and string together into a struct ensures they will never be out of sync